### PR TITLE
Create welcome_email.rst

### PR DESCRIPTION
**Pull request by james-mm**
Welcome email template for end users

### CODE REVIEW

File: source/getting-started/welcome_email.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Hi all,
+ 
+As some of you already know, we are moving to Mattermost as our communication platform. Mattermost is a messaging app where you can talk, share files, and collaborate on projects or initiatives. Mattermost also integrates with many of the apps that you use every day, like **[add apps]**.  
+
+We are moving to Mattermost because it will allow us to become more productive by being able communicate in one place instead of using email, instant chat, and a multitude of other programs.
+
+Some of the major benefits of using Mattermost are:
+
+- Direct 1:1 and group messaging
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Propose linking each point to the appropriate section of the end user docs

File: source/getting-started/welcome_email.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- Channels for topic-based, group-based, or meeting-based chat
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Docs

File: source/getting-started/welcome_email.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- Streamlined collaboration on projects
+
+- Reduced email clutter
+
+- Searching across messages and channels
+
+- Sharing files
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Docs

File: source/getting-started/welcome_email.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+- Direct 1:1 and group messaging
+
+- Channels for topic-based, group-based, or meeting-based chat
+
+- Streamlined collaboration on projects
+
+- Reduced email clutter
+
+- Searching across messages and channels
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Docs

File: source/getting-started/welcome_email.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+- Sharing files
+
+To get started:
+
+1. Open a browser on your computer, go to **[Mattermost URL]** and log in with your **[LDAP/AD, SAML, Google, etc]** credentials. 
+
+2. `Download <https://about.mattermost.com/download/#mattermostApps>`_ the Mattermost apps for desktop and mobile. Refer to the `Mattermost Welcome Guide <https://docs.mattermost.com/help/getting-started/welcome-to-mattermost.html>`_ on how to get up and running quickly.
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Should this step be optional?

File: source/getting-started/welcome_email.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+- Reduced email clutter
+
+- Searching across messages and channels
+
+- Sharing files
+
+To get started:
+
+1. Open a browser on your computer, go to **[Mattermost URL]** and log in with your **[LDAP/AD, SAML, Google, etc]** credentials.
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: We should make it clear that at any time they can click the Mattermost URL to go back to the service. It's not just a one time link for first time users

Comment (esethna) [reply source/getting-started/welcome_email.rst]: Doesn't have to be part of these steps, but it should be mentioned somewhere how to get back to the Mattermost

File: source/getting-started/welcome_email.rst [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+To: End users
+
+Subject: New Communication Platform - Mattermost
+
+
+Hi all,
+ 
+As some of you already know, we are moving to Mattermost as our communication platform. Mattermost is a messaging app where you can talk, share files, and collaborate on projects or initiatives. Mattermost also integrates with many of the apps that you use every day, like **[add apps]**.  
+
+We are moving to Mattermost because it will allow us to become more productive by being able communicate in one place instead of using email, instant chat, and a multitude of other programs.
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Intro seems a bit wordy. Maybe it would be better to use our tag line. Something like,

"Mattermost will host all our communication in one place, is instantly searchable and available from all your devices."

Comment (james-mm) [reply source/getting-started/welcome_email.rst]: Updated

File: source/getting-started/welcome_email.rst (hunk 1 above)
Comment (james-mm) [reply source/getting-started/welcome_email.rst]: The end of the email points to the end user getting started guide, which does this in more detail: https://docs.mattermost.com/help/getting-started/welcome-to-mattermost.html.

File: source/getting-started/welcome_email.rst (hunk 2 above)
Comment (james-mm) [reply source/getting-started/welcome_email.rst]: See above

File: source/getting-started/welcome_email.rst (hunk 4 above)
Comment (james-mm) [reply source/getting-started/welcome_email.rst]: See above

File: source/getting-started/welcome_email.rst (hunk 3 above)
Comment (james-mm) [reply source/getting-started/welcome_email.rst]: See above

File: source/getting-started/welcome_email.rst (hunk 6 above)
Comment (james-mm) [reply source/getting-started/welcome_email.rst]: Updated

File: source/getting-started/welcome_email.rst (hunk 5 above)
Comment (james-mm) [reply source/getting-started/welcome_email.rst]: Desktop and mobile are a big part of our customers' every use case. They can skip if they don't want to download. The welcome guide is essential to them getting up and running quickly on MAttermost.

File: source/getting-started/welcome_email.rst [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+- Reduced email clutter
+
+- Searching across messages and channels
+
+- Sharing files
+
+To get started:
+
+1. Open a browser on your computer, go to **[Mattermost URL]** and log in with your **[LDAP/AD, SAML, Google, etc]** credentials. Remember to bookmark the URL as that you can use it next time to log in.
```
Comment (esethna) [inline source/getting-started/welcome_email.rst]: Remember to bookmark the URL so you can use it to log in next time.
